lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -91,7 +91,9 @@
                              currentSessionId: this.model.isEnrolledInSession() ?
                                  this.model.get('course_run_key') : null,
                              enrollUrl: this.model.get('enroll_url'),
-                             courseHomeUrl: this.model.get('course_url')
+                             courseHomeUrl: this.model.get('course_url'),
+                             expiredAt: this.entitlement.expired_at,
```

Comment:
This should probably be expired_at_datetime instead (which might let you drop the explicit update_expired_at() in views.py, which strikes me as something to avoid if we can).

Reply:
So this comes from the context object, it's not specifically a python property we can just use. We set it in the to_dict() method of CourseEntitlement.

```
expired_at = strftime_localized(self.expired_at_datetime, 'SHORT_DATE') if self.expired_at_datetime else None

        return {
            'uuid': str(self.uuid),
            'course_uuid': str(self.course_uuid),
            'expired_at': expired_at,
            'expiration_date': expiration_date
        }```

Reply:
Ah OK right.  Thanks